source/sass/mofo-bootstrap/overrides/_mixins.scss

```diff
@@ -28,8 +28,7 @@
   }
 
   &:active,
-  &.active,
-  .open > &.dropdown-toggle {
+  &.active {
```

Comment:
There's a whole bunch of "double rules" in this file now.

Reply:
Ah. One is targeting for pseudo class `:active` and one is targeting `.active` class.

There are multiple rulesets for them in this file but each is wrapped in a different mixin.

Reply:
ahhhh